library/class-foundationpress-mobile-walker.php

```diff
@@ -13,7 +13,7 @@
 if ( ! class_exists( 'Foundationpress_Mobile_Walker' ) ) :
 	class Foundationpress_Mobile_Walker extends Walker_Nav_Menu {
 		function start_lvl( &$output, $depth = 0, $args = array() ) {
-			$indent = str_repeat("\t", $depth);
+			$indent  = str_repeat( "\t", $depth );
```

Comment:
Why is there 2 spaces before the equals sign here? Is this so that it lines up with the equals sign on the line below?

Reply:
Yes, just like the change [here](https://github.com/olefredrik/FoundationPress/pull/1178/files#diff-9a4a6b0285479ce4bb5e0a1317cf70a0R50) where it's more obvious. This one though is a little odd because the next line has a dot before the equal sign so they only line up visually, but I thought it maybe was within the standard, so I let it in. It was one of those cases where I could go either way.

Reply:
@Aetles if that's what the codesniffer said then I'm fine with it. I just wanted clarification and the other example you showed definitely confirms that they are lining up the equal signs. Thanks!